boards/nrf51dk/Makefile

```diff
@@ -1,15 +1,26 @@
+# Makefile for building the tock kernel for the nRF development kit
+
+OBJCOPY?=arm-none-eabi-objcopy
+OBJDUMP?=arm-none-eabi-objdump
+OBJDUMP_FLAGS+= --disassemble-all --source --disassembler-options=force-thumb -C --section-headers
+
 .PHONY: all
-all: target/target/release/nrf_pca10028
+all: target/target/release/nrf51dk target/target/release/nrf51dk.lst
```

Comment:
nit: is this the right place for the `.lst` dependency? Feels like that should come along as an artifact of one of the other rules?